controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/ArchiveUriUpdater.java

```diff
@@ -55,9 +57,8 @@ protected double maintain() {
         for (var application : applications.asList()) {
             for (var instance : application.instances().values()) {
                 for (var deployment : instance.deployments().values()) {
-                    tenantsByZone.get(deployment.zone()).add(instance.id().tenant());
-                    applications.decideCloudAccountOf(new DeploymentId(instance.id(), deployment.zone()), application.deploymentSpec())
-                            .ifPresent(account -> accountsByZone.get(deployment.zone()).add(account));
+                    if (zoneRegistry.isEnclave(deployment.cloudAccount())) accountsByZone.get(deployment.zone()).add(deployment.cloudAccount());
+                    else tenantsByZone.get(deployment.zone()).add(instance.id().tenant());
```

Comment:
SO this won't work before all deployemnts are re-done, but I guess that's OK?

Reply:
Yes, there are no exclave deployments currently and that method will return false since the default cloud account is `CloudAccount.empty`, which will result in the old behavior